### PR TITLE
Feature/single user decorator

### PR DESCRIPTION
**Pull request by LoyVanBeek**
Fix issue #2

### CODE REVIEW

File: scripts/telegram_ros_bridge [hunk 1]
```diff
@@ -43,6 +45,7 @@ class TelegramROSBridge(object):
         self._telegram_updater.dispatcher.add_error_handler(
             lambda _, update, error: rospy.logerr("Update {} caused error {}".format(update, error)))
 
+
```
Comment (MatthijsBurgh) [inline scripts/telegram_ros_bridge]:
```suggestion
```

**Review by MatthijsBurgh — COMMENTED** [review]
White line, plus th biggest question: can you still send /start without being active? Otherwise you created a deadlock.

**Comment by LoyVanBeek** [comment]
Of course it still works, I tested it with my own bot API key. 
The ```_telegram_start_callback``` is not decorated with ```@allow_only_active```

**Comment by MatthijsBurgh** [comment]
Why not also wrap the outgoing callbacks? Because if there is no active chat, you can’t send anything over telegram.

**Comment by MatthijsBurgh** [comment]
Maybe not related to this pr, but we have a ‘telegram message’ vs ‘ros string’ callback. Also ‘telegram photo’ vs ‘ros image’ callback. And the order of the member functions is inconsistent. As both the location callbacks have added at the end, while the others are telegram 2x, ros 2x. Would be nice to clean this up.

**Comment by LoyVanBeek** [comment]
As for the outgoing callbacks, there is no one to send this feedback to in case no user is connected. The ROS nodes that send info just publish, it's on an actionserver that might fail. 

As for ordering and method naming: Message and Photo are Telegram concepts (that what their API uses).

**Comment by LoyVanBeek** [comment]
@reinzor can yu plz take a look?

**Comment by reinzor** [comment]
I will look at this next Saturday

**Review by reinzor — APPROVED** [review]
Did some minor refactors, can be merged i.m.o.

**Pull request merged**